.github/workflows/testing.yml

```diff
@@ -467,37 +467,97 @@ jobs:
-         run: mkdir coverage
-
-       - name: Build API.go
-         working-directory: ./api
-         run: go build
-
-       - name: Start API Server
-         working-directory: ./api
-         run: go run api.go &
-
-       - name: Install python dependencies
-         working-directory: ./
-         run: |
-           python3 -m pip install build pytest
-           python3 -m build ./client/
-           python3 -m pip install client/dist/*
-
-       - name: Testing
-         env:
-           API_ADDRESS: "localhost:7878"
-         working-directory: ./client/tests
-         run: pytest
+      - name: Download Working Compiled Directories
+        uses: actions/download-artifact@v3
+        with:
+          name: compiled-workdir
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+
+      - name: Make Coverage Directory
+        working-directory: ./
+        run: mkdir coverage
+
+      - name: Build API.go
+        working-directory: ./api
+        run: go build
+
+      - name: Start API Server
+        working-directory: ./api
+        run: go run api.go &
+
+      - name: Install python dependencies
+        working-directory: ./
+        run: |
+          python3 -m pip install build pytest
+          python3 -m build ./client/
+          python3 -m pip install client/dist/*
+
+      - name: Testing
+        env:
+          API_ADDRESS: "localhost:7878"
+        working-directory: ./client/tests
+        run: pytest
+  schedule:
+    runs-on: ubuntu-latest
+    name: Schedule Jobs
+    environment: Integration testing
+    needs: setup
+    steps:
+      - uses: actions/checkout@v2
+      - name: Start minikube
+        uses: medyagh/setup-minikube@master
+      - name: Set up Postgres deployment on Minikube
+        run: kubectl apply -f coordinator/schedulepod/postgres
+      - name: Set up Redis deployment on Minikube
+        run: kubectl apply -f coordinator/schedulepod/redis
+      - name: Set up etcd deployment on Minikube
+        run: kubectl apply -f coordinator/schedulepod/etcd
+      - name: Get pods in the cluster
+        run: kubectl get pods -A
+      - name: Get deployments in cluster
+        run: kubectl get deployments -A
+      - name: Get services in the cluster
+        run: kubectl get services -A
+      - name: Build images
+        run: |
+          export SHELL=/bin/bash
+          eval $(minikube -p minikube docker-env)
+          docker build . --file runner/Dockerfile --tag featureformcom/worker
+          docker build . --file coordinator/schedulepod/Dockerfile --tag featureformcom/scheduletester
+          echo -n "verifying images:"
+          docker images
+      - name: Deploy to minikube
+        run: kubectl apply -f coordinator/schedulepod/runjob.yaml
+      - name: List jobs
+        run: kubectl get jobs -A
+      - name: Wait until test pod is completed
+        run: kubectl wait --for=condition=complete --timeout=300s job -l app=example
```

Comment:
Is this testing for one pod? We should probably test if the schedule can be updated too right?